src/gemdat/plots/matplotlib/_paths.py

```diff
@@ -77,6 +91,16 @@ def energy_along_path(*, path: Pathway, structure: Structure,
                           rotation=45)
     ax_up.get_yaxis().set_visible(False)
 
+    # If available, plot the other pathways
+    if isinstance(paths, list):
+        for idx, path in enumerate(paths[1:]):
+            if path.energy is None:
+                raise ValueError('Pathway does not contain energy data')
+            ax.plot(range(len(path.energy)),
+                    path.energy,
```

Comment:
Isn't it implied? If you leave out the range, does matplotlib not automatically generate the range?

```suggestion
            ax.plot(path.energy,
```

Reply:
true!